src/remixAppManager.js

```diff
@@ -84,9 +84,23 @@ export class RemixAppManager extends AppManagerApi {
       location: 'mainPanel'
     }
+    var mythx = {
+      name: 'remythx',
+      displayName: 'MythX',
+      description: 'Security analysis of smart contracts by using MythX',
+      events: [],
+      methods: [],
+      notifications: {
+        'solidity': ['compilationFinished']
+      },
+      url: 'http://remythx.xyz',
+      icon: 'http://remythx.xyz/logo.png',
```

Comment:
could we use `https` for both url and icon?

Reply:
yes